docs/variables/valorant.md

```diff
@@ -0,0 +1,153 @@
+---
+id: valorant
+---
+
+# $(valorant)
+
+Returns a range of metadata around a broadcaster's [**Riot Games**](https://www.riotgames.com/en) account, in relation to the game VALORANT, that can be accessed via multiple different variable members for each unique piece of information.

```

Comment:
* The league one links to the league website, should we add that on `VALORANT`?
* Can you change the riot games link to remove the `en` suffix, I think it should let the users' browser prefs take priority then

Reply:
sure, and done